demo_nodes_cpp/test/test_executables_tutorial.py.in

```diff
@@ -17,7 +17,6 @@ import launch_testing_ros
 
 def generate_test_description():
     env = os.environ.copy()
-    env['OSPL_VERBOSITY'] = '8'  # 8 = OS_NONE
```

Comment:
@clalancette unrelated?

Reply:
Yeah, I just noticed it while I was in here.  It's to support Opensplice stuff, and we don't need it anymore.  I can remove it from this PR if you don't think it should be in here.

Reply:
I ended up not including this in the patch I just pushed.  That just keeps this PR focused.  I may do a later cleanup where I remove this from a bunch of places.